index.html

```diff
@@ -18,8 +18,11 @@ <h2>Our Advice for Newbies!</h2>
     <a href="advice/angela-rivera-advice.html" target="_blank">Angela Rivera's Advice 💙</a><br>
     <a href="advice/aileen-cm-advice.html">Aileen CM's advice!🇵🇷</a><br>
     <a href="advice/destiny1s-advice.html">🤍Destiny's Advice🤍</a><br>
-    <a href="advice/torres-ml-advice.html" target="_blank"
```

Comment:
HI @Luilu27,

Congrats on issuing your first pull request! That's awesome!!

It looks like the styling Mari added to the link to her advice page has been removed. Can you add it back? (It's the link just above your link.)

Also, there are two added <i> tags. Is this something you want added to your link? If so, can you add it between the anchor tags `<a></a>` of the link to your advice page?

Just in case you're adding Font Awesome icons, since there isn't a link to Font Awesome in the <head> of the index.html file, the icon isn't going to render. If you would like to add a Font Awesome icon to your advice page - that's perfectly fine! You can add the link to Font Awesome in the <head> of _your_ advice file and add a Font Awesome icon to the content of your page. (Or you can add an emoji instead like some other Skillcrushers have done.)

Once you've made those two changes, add and commit them. Then push the advice branch to GitHub. It will automatically update your pull request and we'll take a look!

Reach out if you have any questions!

Best,
Lisa